packages/world/src/World.sol

```diff
@@ -40,7 +40,7 @@ contract World is StoreKernel, IWorldKernel {
   using WorldResourceIdInstance for ResourceId;
 
   /// @notice Address of the contract's creator.
-  address public immutable creator;
+  address public creator;
```

Comment:
😭

Reply:
I know! We need this as a storage variable so it can be read/written by the proxy